### PR TITLE
PAI overlay on PDAs

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds a small overlay when pais are inserted into your pda

image

image
## Why It's Good For The Game
This was another legacy feature that was removed with modpdas, and other servers (like bee, though i personally dont agree with the creator.) have it already. sooooo why not make one for tg
## Changelog
:cl:
imageadd: indicator when a pda contains a pai
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/computers/item/computer.dm [hunk 1]
```diff
@@ -204,6 +204,7 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 		user.put_in_hands(inserted_pai)
 		balloon_alert(user, "removed pAI")
 		inserted_pai = null
+		update_overlays()
```
Comment (Mothblocks) [inline code/modules/modular_computers/computers/item/computer.dm:207]: You're meant to call `update_appearance(UPDATE_ICON)`, `update_overlays` is a pure proc

File: code/modules/modular_computers/computers/item/computer.dm [hunk 2]
```diff
@@ -416,6 +417,7 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 		inserted_pai = null
 	if(inserted_disk == gone)
 		inserted_disk = null
+	update_overlays()
```
Comment (Mothblocks) [inline code/modules/modular_computers/computers/item/computer.dm:420]: Only do this if inserted_pai is removed

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/computers/item/computer.dm (hunk 1 above)
Comment (etherware-novice) [reply code/modules/modular_computers/computers/item/computer.dm:207]: whats a pure proc?

Comment (Mothblocks) [reply code/modules/modular_computers/computers/item/computer.dm:207]: It shouldn't have side effects on its own, it just returns the new overlays

**Pull request merged**